lib/ember-cli/app.rb

```diff
@@ -95,10 +95,14 @@ def ember_path
 
     private
 
-    delegate :match_version?, :non_production?, to: Helpers
+    delegate :match_version?, to: Helpers
     delegate :configuration, to: EmberCLI
     delegate :tee_path, :npm_path, :bundler_path, to: :configuration
 
+    def non_production?
+      !Rails.env.production?
```

Comment:
@rwz yes.

I've toggled that setting to false in the project, but I'm not sure if it'll be accepted.

Either way, we might want to clarify how we determine the Ember ENV?